packages/cli-next/package.json

```diff
@@ -47,7 +47,7 @@
     "typescript": "4.9.5"
   },
   "dependencies": {
-    "@novu/shared": "^0.23.1",
+    "@novu/shared": "workspace:",
     "@novu/studio": "^0.0.1-alpha.87",
```

Comment:
does `@novu/studio` also need to be `"workspace:"`?

Reply:
It's still living in the other repository, so this one stays for now. We'll be modifying the CLI to remove this dependency when we create Studio in Web.